### PR TITLE
org: Transition inactive maintainers to Emeritus

### PR DESCRIPTION
**Pull request by howardjohn**
This PR moves maintainers who appear to be inactive in the project to Emeritus.

This has a few motivations:
* Thank and honor all the past maintainers for all the invaluable work they have done over the years
* Trim the list of maintainers to (1) improve discover-ability of who is able to actively maintainer/review/etc an area and (2) reduce the number of privileged users in the org

This list was compiled by examining the maintainers list activity (PRs, issues, comments, etc) over the past 1-2 years. Note that https://github.com/istio/community/pull/1319 attempts to formally define a criteria for what "inactive" constitutes. In some cases, I used my own judgement for folks I believe are no longer interested in being maintainers due to more recent shifts in priorities, so a few individuals do have activity within the last year.

The goal here is very much not to "kick people out" of the maintainers list. If you feel you are interested in continuing your maintainer role, simply comment here (or DM me) and I will update the PR. I will also give this PR a while before merging to give people time to see it. However, one note: you do not need to hold a formal title of "maintainer" to be a valuable contributor to the project, and if you do step down the process to step back into the role when you have more time to commit to Istio is fairly quick and easy.

Tagging impacted members: @Monkeyanator @morvencao @ostromart @mandarjog @litong01 @lambdai @JimmyCYJ @john-a-joyce @incfly @esnible @bianpengyuan

Thanks again for all your contributions!

Closes https://github.com/istio/community/issues/1262
Past PRs: https://github.com/istio/community/issues/817

### CODE REVIEW

**Review by linsun — COMMENTED** [review]
LGTM

the list looks right to me

**Comment by kfaseela** [comment]
LGTM

**Review by esnible — APPROVED** [review]

**Review by morvencao — DISMISSED** [review]
thanks!
/lgtm

File: org/teams.yaml [hunk 1]
```diff
@@ -355,19 +325,17 @@ teams:
...
-      - myidpt
-      - nrjpoddar
-      - stevenctl
+      - costinm # Env
+      - ericvn # T&R
+      - GregHanson # PSWG
+      - howardjohn # Control Plane
+      - justinpettit # PSWG
+      - lei-tang # Security, E&T
+      - lizan # Data Plane, Security
```
Comment (zirain) [inline org/teams.yaml:334]: Is lizan still active? should we nominate a new person? @kyessenov should be the guy if he want.

Comment (hzxuzhonghu) [reply org/teams.yaml:336]: cc @lizan

**Comment by howardjohn** [comment]
I will merge tomorrow if there are no objections.

**Review by kfaseela — APPROVED** [review]

File: org/teams.yaml (hunk 1 above)
Comment (craigbox) [reply org/teams.yaml:336]: @zirain can you please ping @lizan internally and ask him to make a decision here?

Comment (zirain) [reply org/teams.yaml:336]: IIRC, @lizan leave Tetrate for a while?

Comment (craigbox) [reply org/teams.yaml:336]: /cc @therealmitchconnors as Lizan is now at Aviatrix.

Comment (craigbox) [reply org/teams.yaml:336]: /cc @therealmitchconnors as Lizan is now at Aviatrix.

**Pull request merged**